src/hooks/useSource.js

```diff
@@ -1,7 +1,9 @@
 import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
 
-export const useSource = (id) => {
-    const source = useSelector(({ providers }) => providers.entities.find(source => source.id  === id));
+export const useSource = (sourceId) => {
```

Comment:
Firstly. Nice work on the hook. Have one comment here. I think that it would be safer and less magical to make the `sourceId` required param and ditch the `useParam`. You are playing with potential side effects which might bite you later on.

Reply:
That's the point of this PR. I was using `useParams` and `useSource` in all components together, so this PR just removes unnecessary duplication across the components (and less imports). 

Right now, the purpose of `useSource` is to get the active source (which is being edited, removed, etc.) by url... actually, the `sourceId` is not used anywhere. :D

Reply:
Ok so in that case, I would remove the param and have only one source of the ID.

Reply:
Okay, will do it!

Reply:
Removed